src/js/jsx/tools/SuperselectOverlay.jsx

```diff
@@ -210,11 +220,24 @@ define(function (require, exports, module) {
             svg.selectAll(".superselect-bounds").remove();
             svg.selectAll(".superselect-marquee").remove();
             svg.selectAll(".artboard-adder").remove();
+            svg.selectAll(".vector-mask-hud").remove();
 
             if (!currentDocument || this.state.modalState) {
                 return null;
             }
 
+            this._hudGroup = svg.insert("g", ".hud-group")
```

Comment:
Maybe this can be in the if statement below. Otherwise we're creating an unnecessary "vector-mask-hud" `<g>` every time we redraw.